src/parse/parsers.ts

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+import parseFrenchNationalId from './frenchNationalId';
+import parseSwissDrivingLicense from './swissDrivingLicense';
+import parseTD1 from './td1';
+import parseTD2 from './td2';
+import parseTD3 from './td3';
+
+export default {
```

Comment:
same as in index, why export default?

Reply:
it was export default in the provious version

Reply:
no, it was `module.exports = {}` it's not the same as export default.
In the new version we have only a `default` key in the object when we require the package:
```
> Object.keys(require('mrz'))
[ 'default' ]
```

Reply:
aaaa okey i will open a new PR and fix it

Reply:
yes, please